core/modules/node/node.module

```diff
@@ -1876,12 +1882,14 @@ function node_block_view($delta = '', $settings = array()) {
 
   switch ($delta) {
     case 'syndicate':
-      $block['subject'] = t('Syndicate');
-      $block['content'] = array(
-        '#theme' => 'feed_icon',
-        '#url' => 'rss.xml',
-        '#title' => t('Syndicate'),
-      );
+      if (backdrop_valid_path('rss.xml')) {
```

Comment:
Now that we're always keeping rss.xml, we don't need this if statement any more I don't think. But it does no harm.